test/runtests.jl

```diff
@@ -2,17 +2,12 @@ using Test
 using StaticArrays
 using RootedTrees
 
-trees_array = (RootedTree([1,2,3]),
-               RootedTree([1,2,3]),
-               RootedTree([1,2,2]),
-               RootedTree([1,2,3,3]))
+trees_array = (rootedtree([1,2,3]),
+               rootedtree([1,2,3]),
+               rootedtree([1,2,2]),
+               rootedtree([1,2,3,3]))
 
-trees_marray = (RootedTree(@MArray [1,2,3]),
```

Comment:
May I ask why you've removed these additional tests? I wanted to make sure that everything works also for non-standard arrays.

Reply:
The problem is that `MArray` is not type-stable if you do range indexing.
```julia
julia> using StaticArrays

julia> v = @MArray([1,2,3])
3-element MArray{Tuple{3},Int64,1,3} with indices SOneTo(3):
 1
 2
 3

julia> v[1:2]
2-element Array{Int64,1}:
 1
 2
```

Supporting `MArray` is hard and it will run very slowly anyway, so I think it would be better to just drop it.